src/main/java/edu/whimc/observations/utils/sql/migration/schemas/Schema_4.java

```diff
@@ -0,0 +1,25 @@
+package edu.whimc.observations.utils.sql.migration.schemas;
+
+import edu.whimc.observations.utils.sql.migration.SchemaVersion;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class Schema_4 extends SchemaVersion {
+
+    private static final String ADD_CATEGORY =
+            "ALTER TABLE whimc_observations ADD COLUMN observation_color_stripped TEXT;";
+
+    public Schema_4() {
+        super(4, null);
+    }
+
+    @Override
+    protected void migrateRoutine(Connection connection) throws SQLException {
+        try (PreparedStatement statement = connection.prepareStatement(ADD_CATEGORY)) {
+            statement.execute();
```

Comment:
Would be good to also modify the existing rows to contain the color-stripped observation.

Reply:
Still need to test my code for this want to do it locally before manipulating the actual observations table.

Reply:
Create some observations using an older version on your test server and set the value inside `.schema_version` to `3` to have the migration run again.

I wouldn't put this plugin on the beta server until you're confident with the SQL changes on your local server.